### PR TITLE
Unit-tests with CI Update

### PR DESCRIPTION
**Pull request by jakebolewski**
closes #31

### CODE REVIEW

**Comment by charleskawczynski** [comment]
Supersedes #51

File: bors.toml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 status = [
-  "dockerci (1.4, ubuntu-latest)",
+  "test (ubuntu-latest)',
+  'test (windows-latest)',
+  'test (macos-latest'),
```
Comment (charleskawczynski) [inline bors.toml:4]: @jakebolewski bors doesn't like commas after the last entry

**Comment by jakebolewski** [comment]
@charleskawczynski I don't think I have the permissions to disable the appveyor integration

**Comment by jakebolewski** [comment]
Ok let's see if this works...

**Comment by charleskawczynski** [comment]
> I don't think I have the permissions to disable the appveyor integration

I can unhook it

**Comment by charleskawczynski** [comment]
Actually, bors doesn't depend on appveyor, so it shouldn't hold anything up

**Comment by jakebolewski** [comment]
:fist_oncoming: powershell

https://github.com/MicrosoftDocs/PowerShell-Docs/issues/2361

**Comment by jakebolewski** [comment]
Yay! https://github.com/JuliaComputing/MKL.jl/pull/35

**Comment by jakebolewski** [comment]
:disappointed: https://github.com/JuliaPy/PyPlot.jl/issues/315#issuecomment-645533629

**Pull request merged**